setup.py

```diff
@@ -4,7 +4,13 @@
     name='pr_pilot',
     version='1.0.0',
     packages=find_packages(),
-    install_requires=[],
+    install_requires=[
+        'pydantic==2.7.0',
+        'urllib3==2.2.1',
+        'python-dateutil==2.9.0',
+        'wheel==0.42.0',
+        'setuptools==69.1.1',
+    ],
```

Comment:
/pilot This list should be generated in the script by reading `requirements.txt`